aiopg/utils.py

```diff
@@ -231,3 +233,10 @@ def __exit__(self, *args):
         coroutines._COROUTINE_TYPES += (_ContextManager,)
     except:
         pass
+
+
+# map psycopg2cffi to psycopg2, so any code
+# that imports psycopg2 will use psycopg2cffi
+if PY_IMPL == 'PyPy':
+    from psycopg2cffi import compat
+    compat.register()
```

Comment:
Hmm. 
Thanks @jettify for pointing on.
Maybe better to explicitly import either `psycopg2` or `psycopg2cffi`?
Doing this we'll never catch an import order problem.